hack/build-and-push.sh

```diff
@@ -54,26 +62,24 @@ for TASK in $SCRIPTDIR/../tasks/*.yaml; do
     yq -M e ".spec.steps[] |= select(.image == \"appstudio-utils\").image=\"$APPSTUDIO_UTILS_IMG\"" \
         $TASK >> $TASK_TEMP
     echo --- >> $TASK_TEMP
-    REF="$APPSTUDIO_TASKS_REPO:$BUILD_TAG-$PART"
+    REF="$APPSTUDIO_TASKS_REPO:${TEST_REPO_NAME:+tasks-}$BUILD_TAG-$PART"
     for file in $PIPELINE_TEMP/*.yaml; do
         yq e -i "(.spec.tasks[].taskRef | select(.name == \"$TASK_NAME\")) |= {\"name\": \"$TASK_NAME\", \"bundle\":\"$REF\"}" $file
         yq e -i "(.spec.finally[].taskRef | select(.name == \"$TASK_NAME\")) |= {\"name\": \"$TASK_NAME\", \"bundle\":\"$REF\"}" $file
     done
     COUNT=$((COUNT+1))
 done
-echo Creating $APPSTUDIO_TASKS_REPO:$BUILD_TAG-$PART
-tkn bundle push -f $TASK_TEMP $APPSTUDIO_TASKS_REPO:$BUILD_TAG-$PART
-
-# Build Pipeline budle with pipelines pointing to newly built appstudio-tasks
-PIPELINE_BUNDLE=quay.io/$MY_QUAY_USER/build-templates-bundle:$BUILD_TAG
-tkn bundle push $PIPELINE_BUNDLE -f ${PIPELINE_TEMP}/base.yaml
 
-HACBS_BUNDLE=quay.io/$MY_QUAY_USER/hacbs-templates-bundle:$BUILD_TAG
-tkn bundle push $HACBS_BUNDLE -f ${PIPELINE_TEMP}/hacbs.yaml
+APPSTUDIO_TASKS_IMG=$APPSTUDIO_TASKS_REPO:${TEST_REPO_NAME:+tasks-}$BUILD_TAG-$PART
+echo "Creating $TASK_TEMP $APPSTUDIO_TASKS_IMG"
+tkn bundle push -f $TASK_TEMP $APPSTUDIO_TASKS_IMG
 
-HACBS_CORE_BUNDLE=quay.io/$MY_QUAY_USER/hacbs-core-service-templates-bundle:latest
-tkn bundle push $HACBS_CORE_BUNDLE -f ${PIPELINE_TEMP}/hacbs-core-service.yaml
+# Build Pipeline bundle with pipelines pointing to newly built appstudio-tasks
+tkn bundle push $PIPELINE_BUNDLE_IMG -f ${PIPELINE_TEMP}/base.yaml
+tkn bundle push $HACBS_BUNDLE_IMG -f ${PIPELINE_TEMP}/hacbs.yaml
+tkn bundle push $HACBS_CORE_BUNDLE_IMG -f ${PIPELINE_TEMP}/hacbs-core-service.yaml
 
 if [ "$SKIP_INSTALL" == "" ]; then
-    $SCRIPTDIR/util-install-bundle.sh $PIPELINE_BUNDLE
+    : "${INSTALL_BUNDLE_NS:=build-templates}"
```

Comment:
you can remove this line, not sure what it does, but basically when `INSTALL_BUNDLE_NS` is not passed to the script then it's empty and util-install-bundle.sh will be using default.

Reply:
It's a shorthand for `INSTALL_BUNDLE_NS=${INSTALL_BUNDLE_NS:-build-templates}`. But it makes sense to remove it considering the comments in [util-install-bundle](https://github.com/redhat-appstudio/build-definitions/pull/57/files/a31d98d16dda3b9b2d25ffedad32caf0c37790b9#diff-0cf40ce7cea04e5856c097541c8f9bd5b99c0e888ef7e78e43e37304ca9954b2) script